similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
Then why do we keep this file "configurations.properties" (and we don't move everything here to "SimConfigurator")?